### PR TITLE
Read push config json

### PR DESCRIPTION
**Pull request by edewit**
I've added a `LoadConfigJson` method so one can read the config object and then add dynamic properties afterwards.

fixes [AGPUSH-1432](https://issues.jboss.org/browse/AGPUSH-1432)

### CODE REVIEW

**Comment by matzew** [comment]
This still enables a user to update the categories during runtime, right ?

**Comment by edewit** [comment]
@matzew yeah example:

```csharp
Registration registration = new WnsRegistration();
PushConfig config = registration.LoadConfigJson("push-config.xml");
config.Categories = //user categories
registration.Register(config);
```
or if you don't have any dynamic alias or categories:

```csharp
Registration registration = new WnsRegistration();
registration.Register();
```

**Comment by matzew** [comment]
@edewit that;s awesome. thanks

File: test/ConfigReadTest.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
```
Comment (matzew) [inline test/ConfigReadTest.cs]: wanna move this import a bit down?

**Comment by matzew** [comment]
besides my comment -> :+1:

**Comment by sebastienblanc** [comment]
Myabe we can update the readme.md with your example ? Beside that +1

**Pull request merged**